tests/API/ApiTests.php

```diff
@@ -1,12 +1,14 @@
 <?php
 namespace Auth0\Tests\API;
 
+use Auth0\Tests\Traits\ErrorHelpers;
 use Auth0\SDK\API\Helpers\TokenGenerator;
```

Comment:
I think I mentioned this in a comment above, but this token generator seems to be only used on this test. Please move it to the `/Test/` folder so users don't use it

Reply:
See above